docs/reference/styles/style-selector-syntax.md

```diff
@@ -81,18 +81,20 @@ Selects a control with the specified style class or classes. Multiple classes ar
 
 ```xml
 <Style Selector="Button:focus">
+<Style Selector="Button:focus:pointerover">
 <Style Selector="Button.large:focus">
 ```
 
 
 ```csharp title='C#'
 new Style(x => x.OfType<Button>().Class(":focus"));
+new Style(x => x.OfType<Button>().Class(":focus").Class(":pointerover"));
 new Style(x => x.OfType<Button>().Class("large").Class(":focus"));
```

Comment:
There appears to be a missing colon `:` with `large`. Might want to fix that here too.

Reply:
Nevermind... I actually read the docs. Will delete this.